compiler/src/dotty/tools/repl/Rendering.scala

```
@@ -70,21 +68,20 @@ private[repl] class Rendering(compiler: ReplCompiler,
     d.symbol.showUser
 
   /** Render value definition result */
-  def renderVal(d: Denotation)(implicit ctx: Context): Option[String] = {
+  def renderVal(d: Denotation)(implicit ctx: Context): Try[Option[String]] = {
     val dcl = d.symbol.showUser
 
-    try {
+    Try {
       val resultValue =
         if (d.symbol.is(Flags.Lazy)) Some("<lazy>")
         else valueOf(d.symbol)
 
       resultValue.map(value => s"$dcl = $value")
     }
-    catch { case ex: InvocationTargetException => Some(renderError(ex)) }
```

We should propagate any exception that is not an `InvocationTargetException`